File: days_mobile/lib/models/Classroom.dart
class Classroom {
  int id;
  String department;
  int floor;
  int number;

  Classroom(
      {this.id,
      this.department,
      this.floor,
      this.number,});

  Classroom.fromJson(Map<String, dynamic> json) {
    id = json['id'];
    department = json['department'];
    floor = json['floor'];
    number = json['number'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['id'] = this.id;
    data['department'] = this.department;
    data['floor'] = this.floor;
    data['number'] = this.number;
    
    return data;
  }
}

File: days_admin/app/Models/Classroom.php
<?php

namespace App\Models;

use App\Http\Traits\WithImageUpload;
use Illuminate\Database\Eloquent\Model;


class Classroom extends Model
{
    use WithImageUpload;

    protected $table = "classrooms";

    

    protected $fillable = [
        'department',
		'floor',
		'number',
		
    ];

    protected $dates = [
        'created_at'
    ];

    
    


}
